### DISCUSSION TITLE
Processo de acompanhamento e mentoria durante o desenvolvimento dos projetos

### DISCUSSION THREAD

**Discussion by Marlysson** [frontendbr/forum#1027] [Discussão]
Boa noite pessoal.. Gostaria de levantar um debate a título de curiosidade..

No local de trabalho de vocês como é o processo de mentoria e acompanhamento dos desenvolvedores que estão adentrando nos projetos?

Há desenvolvedores disponíveis para tirar dúvidas, acompanhar para ver se está sendo evoluído.. explicar as regras.. aconselhar em algum ponto que não está sendo feito corretamente..

Há uma cultura de evolução dos funcionário por meio de revisão de código para não ficar naquilo de "entrega,funciona e bola pra frente.."

Ou mandam as tasks e fica meio que na base do "se vira e entrega" ? :smile: 

Enfim.. seria interessante saber como funciona nos lugares..

**Comment by Cleysonlb** [comment]
Onde trabalho não tem nada muito formal, mas temos em mente que somos um time, e com isso não temos receio de perguntar ou tirar duvidas, todos são mt solicitos.
Falando um pouco sobre mentoria/acompanhamento, todo código passa por code review com aprovação de pelo menos 2 membros do time.
Realizamos retrospectiva toda sexta para descrever como foi a semana e tocar em pontos de melhoria, além das plannings e das reuniões diarias.

**Comment by tunnes** [comment]
Bom mesmo trabalhando remotamente nosso time tem o hábito de te implementar tarefas alternando os pares de acordo com a especialidade individual.. 
Isso não era uma regra nasceu pela intimidade e vontade de compartilhar e interagir do pessoal, uma boa para "forçar" o nascimento desse tipo de interação  é ter algo como um planning poker, o code review ou a [DS](http://www.desenvolvimentoagil.com.br/scrum/daily_scrum). Isso além de organizar o fluxo de trabalho acaba meio que por tabela unindo a equipe.
